example/example.client.ts

```diff
@@ -446,6 +446,30 @@ export class ExpressZodAPIClient {
       ),
     );
   }
+  public subscribe<
+    K extends Extract<Request, `get ${string}`>,
+    R extends Extract<PositiveResponse[K], { event: string }>,
+  >(request: K, params: Input[K]) {
+    const path = request.split(/ (.+)/, 2)[1] as Path;
```

Comment:
1. this should probably be extracted into a private method, since it repeats implementation in `provide()`
2. there should also be possible path params substitution

Reply:
extracted in 65cd7c25

Reply:
proposed method
```ts
private substitute(path: string, params: Record<string, any>) {
    return Object.keys(params).reduce<[typeof path, typeof params]>(([final, remains], key) => {
      if (final.includes(`:${key}`)) final = final.replace(`:${key}`, params[key]);
      else remains[key] = params[key];
      return [final, remains];
    }, [path, {}]);
  }
```

Reply:
alternative implementation without reducer having less types
```ts
private substitute(path: string, params: Record<string, any>) {
    const rest = {...params};
    for (const key in params)
      path = path.replace(`:${key}`, () => { delete rest[key]; return params[key] });
    return [path, rest] as const;
  }
```